### ISSUE TITLE
Implement linear regression

### ISSUE THREAD

**Issue by strikeraryu** [DSC-Bennett-University/AI-ML-Starter#2]
## Implement linear regression from scratch in a proper class format.

With function like
* fit() -> fit the model based on the input points and output points
* predict() -> predict using input data points.

you can more functions based on your requirement. 

#### Path  /algorithm

**Comment by GuND0Wn151** [comment]
Implementation of Linear regression using sklearn LinearRegression

The Repo Link is:
https://github.com/GuND0Wn151/Machine_learning

 The Dataset is also provided in the repo

**Comment by strikeraryu** [comment]
No Ml libraries should be used, it should be from scratch using gradient descent

**Comment by m-abdollahi** [comment]
hi @strikeraryu 
can you assign it to me ?

**Comment by strikeraryu** [comment]
ok you can work on it but be sure it is proper class implementation
